spec/fixtures/Project/None/TestProject.xcodeproj/project.pbxproj

```diff
@@ -405,6 +405,7 @@
 				276A2C831A7C1683004BCC6F /* Release */,
 			);
 			defaultConfigurationIsVisible = 0;
+			defaultConfigurationName = Release;
```

Comment:
Hmm, I was weary about this. It's not actually put into projects made via Xcode by default. Only Xcodeproj adds it when you save the project via Xcodeproj.